src/lib/Importer/SchemaImporter.php

```diff
@@ -161,6 +162,10 @@ private function addSchemaTableColumns(Table $table, array $columnList): void
             if (isset($columnConfiguration['nullable'])) {
                 $column->setNotnull(!$columnConfiguration['nullable']);
             }
+
+            if (isset($columnConfiguration['index'])) {
+                $this->addIndexToColumn($columnConfiguration['index'], $location . '.index', $table, $column->getName());
```

Comment:
Nitpick: this one throws `\Ibexa\Contracts\DoctrineSchema\Exception\InvalidConfigurationException`, should be reported on the method PHPDoc.